charts/nextcloud/1.6.31/templates/deployment.yaml

```diff
@@ -182,6 +197,20 @@ spec: {{ include "common.deployment.common_spec" . | nindent 2 }}
                   ffmpeg || echo "Failed to install ffmpeg"
                   echo "Finished."
         {{- end }}
+        {{- if .Values.nextcloud.install_smbclient }}
+        lifecycle:
+          postStart:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  echo "Installing smbclient..."
+                  apt update && \
+                  apt install -y --no-install-recommends \
+                  smbclient || echo "Failed to install smbclient"
+                  echo "Finished."
+        {{- end }}
```

Comment:
Only 1 postStart can exist. same with probes above.

Script must be merged with additional logic to handle both packages